src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+  font-weight: normal;
+}
+
+p {
+  margin: 0;
+}
+
+.link {
+  color: green;
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.heading {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.preamble {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.article {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.